teensy4-rt/memory.x

```diff
@@ -45,8 +45,7 @@ SECTIONS
         KEEP(*(.boot.ivt));
         KEEP(*(.boot.data));
         KEEP(*(.boot.reset));
-        KEEP(*(.boot.init_data));
-        KEEP(*(.boot.zero_bss));
+        KEEP(*(.boot.start));
```

Comment:
Doesn't actually need to be allocated in flash. Since the reset handler will set up ITCM before calling `_start()`, we can rely on `_start()` being available before it's called.